backend/app/Rules/StyleCriteriaCount.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Rules;
+
+use App\Models\StyleCriteria;
+
+class StyleCriteriaCount
+{
+    //TODO: Fetch this from configuration somewhere.
+    public $maxCriteriaCount = 6;
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $_
+     * @param  mixed  $publicationId
+     * @param  array $params
+     * @return bool
+     */
+    public function checkCount($_, $publicationId, $params)
+    {
+        if (!empty($params)) {
+            [$criteriaId] = $params;
+        }
+
+        if ($this->maxCriteriaCount == 0) {
+            //There is no limit, go nuts.
+            return true;
+        }
```

Comment:
Since style criteria are a core opinion of CCR I don't see ambiguity here. 0 style criteria would be breaking our opinion. 

I'm also not sure if this should even be a user configured value. We'll need to encourage a philosophical consideration re FPR and if a focused number of style criteria is beneficial to success of the concept.

Reply:
Is this the opinion?

**Allowed**

* A limited set of style criteria
* An unlimited set of style criteria

**Not Allowed**

* No style criteria

Reply:
Existence of style criteria are a core opinion.... yes...

Limited vs unlimited have informally been decided as a limit of 6.  

The code supports other options in a descriptive way.  It remains to be seen if that is elevated to be configurable for each installation.